tecken/settings.py

```diff
@@ -166,8 +168,13 @@ class Core(AWS, Celery, S3, Configuration):
         },
     ]
 
-    STATIC_ROOT = values.Value(default=os.path.join(BASE_DIR, "frontend/build"))
-    STATIC_URL = "/"
+    MEDIA_URL = "/media/"
+
+    # Root directory for frontend files like index.html
+    FRONTEND_ROOT = values.Value(default=os.path.join(BASE_DIR, "frontend/build/"))
+
+    STATIC_ROOT = values.Value(default=os.path.join(BASE_DIR, "frontend/build/static/"))
+    STATIC_URL = "/static/"
```

Comment:
`MEDIA_URL` can't be a subdirectory of `STATIC_URL`. I fixed that and then had to fix a bunch of things that relied on `STATIC_URL` being `/`.